thirdparty_builtin/CMakeLists.txt

```diff
@@ -68,11 +68,16 @@ if(ENABLE_TESTS)
 
     if(ENABLE_BENCHMARKS)
         ## google benchmark support
-        add_subdirectory(gbenchmark)
+        add_subdirectory(gbenchmark-release-1.1.0)
+
+        if (NOT WIN32)
+          find_library(RT_LIBRARIES rt)
+          set(rt_libraries rt)
```

Comment:
should we use `RT_LIBRARIES` directly? It seems we are simply overriding to use the string "rt"

I don't think we need rt on osx, here is what I used when I needed for conduit :

https://github.com/LLNL/conduit/blob/master/src/libs/relay/CMakeLists.txt